cartofacade/internal/capi/testhelpers.go

```diff
@@ -0,0 +1,48 @@
+package capi
+
+import (
+	"os"
+)
+
+// GetTestConfig gets a sample config for testing purposes.
+func GetTestConfig(sensor string) (CartoConfig, string, error) {
+	dir, err := os.MkdirTemp("", "slam-test")
```

Comment:
what deletes the directory? Do we need to do that manually? If so, should all these places where we use it be defer deleting the directory?

Reply:
os.RemoveAll(dir) will be removing the temp directories